public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -1,25 +1,23 @@
 include ../../../../_includes/_util-fns
 :marked
-  **Dependency Injection** is an important application design pattern.
-  Angular has its own dependency injection framework and
+  **Dependency injection** is an important application design pattern.
```

Comment:
Dependency Injection is a name of a pattern not an action. Shouldn't this be written with uppercase as it was? https://de.wikipedia.org/wiki/Dependency_Injection